egs/swbd/s5c/RESULTS

```diff
@@ -156,6 +156,14 @@ exit 0
 %WER 12.1 | 1831 21395 | 89.1 7.1 3.8 1.3 12.1 48.1 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_12_0.0/eval2000_hires.ctm.swbd.filt.sys
 %WER 13.6 | 1831 21395 | 87.9 8.2 3.9 1.5 13.6 51.0 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
 
+# results with nnet3 cnn+tdnn: local/nnet3/run_tdnn.sh (1.2.2016) (2 epoch training on speed-perturbed and volume perturbed data)
```

Comment:
Minor deviations in results across different runs of the experiment are
common. How different are your results.

On Thu, Mar 3, 2016 at 2:24 AM, tomkocse <notifications@github.com> wrote:

> In egs/swbd/s5c/RESULTS
> <https://github.com/kaldi-asr/kaldi/pull/530#discussion_r54844100>:
>
> > @@ -156,6 +156,14 @@ exit 0
> >  %WER 12.1 | 1831 21395 | 89.1 7.1 3.8 1.3 12.1 48.1 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_12_0.0/eval2000_hires.ctm.swbd.filt.sys
> >  %WER 13.6 | 1831 21395 | 87.9 8.2 3.9 1.5 13.6 51.0 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
> >
> > +# results with nnet3 cnn+tdnn: local/nnet3/run_tdnn.sh (1.2.2016) (2 epoch training on speed-perturbed and volume perturbed data)
>
> The result that i got with basic TDNN is not as good as the one listed in
> the RESULTS, does it matter ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/530/files#r54844100>.
>